wazo_calld/helpers/confd.py

```diff
@@ -133,7 +133,9 @@ def from_id(cls, conference_id, confd_client):
 
 def get_user_voicemail(user_uuid, confd_client):
     try:
-        return confd_client.users.relations(user_uuid).get_voicemail()
+        return confd_client.voicemails.list(user_uuid=user_uuid)['items'][0]
```

Comment:
I've not tested, but I think is not working: `user_uuid=user_uuid`
Why not using `confd_client.users.get(user_uuid)['voicemail']` ?

Reply:
Indeed that looks better :)

Reply:
And I also confirm, that you cannot filter by `user_uuid` on the voicemail resource :)